src/constants/addresses.ts

```diff
@@ -156,3 +156,15 @@ export const PT_PRIZE_POOL_ADDRESSES = {
 export const FIATDAO_WSOHM_ADDRESSES = {
   [NetworkId.MAINNET]: "0xe98ae8cD25CDC06562c29231Db339d17D02Fd486",
 };
+
+export const BALANCER_VAULT_ADDRESSSES = {
+  [NetworkId.MAINNET]: "0xba12222222228d8ba445958a75a0704d566bf2c8",
+};
+
+export const OHM_FRAX_GUNI_LP_ADDRESSSES = {
+  [NetworkId.MAINNET]: "0x61a0C8d4945A61bF26c13e07c30AF1f1ca67b473",
+};
+
+export const OHM_DAI_WETH_BALANCER_LP_ADDRESSSES = {
+  [NetworkId.MAINNET]: "0xc45d42f801105e861e86658648e3678ad7aa70f9",
+};
```

Comment:
I've removed `OHM_FRAX_GUNI_LP_ADDRESSSES` and `OHM_DAI_WETH_BALANCER_LP_ADDRESSSES` since they are now dynamic, however I've kept the balancer vault for the same reason as the previous comment: it looks like Balancer uses a single vault for all of its pools? Correct me if i'm wrong though